File: backend/clear-data.py
#!/usr/bin/env python3
import sqlite3
import os

db_path = './data/pos.db'

try:
    conn = sqlite3.connect(db_path)
    cursor = conn.cursor()

    print('Deleting all data from order_items...')
    cursor.execute('DELETE FROM order_items')

    print('Deleting all data from orders...')
    cursor.execute('DELETE FROM orders')

    conn.commit()
    conn.close()

    print('✓ All data cleared successfully')
except Exception as e:
    print(f'Error clearing data: {e}')
    exit(1)

File: backend/clear-data.js
import Database from 'better-sqlite3';
import { dirname } from 'path';
import { fileURLToPath } from 'url';

const __dirname = dirname(fileURLToPath(import.meta.url));
const dbPath = './data/pos.db';

try {
  const db = new Database(dbPath);

  console.log('Deleting all data from order_items...');
  db.prepare('DELETE FROM order_items').run();

  console.log('Deleting all data from orders...');
  db.prepare('DELETE FROM orders').run();

  console.log('✓ All data cleared successfully');
  db.close();
  process.exit(0);
} catch (error) {
  console.error('Error clearing data:', error);
  process.exit(1);
}
